src/RealtimeChannel.ts

```diff
@@ -406,13 +406,18 @@ export default class RealtimeChannel {
   ): RealtimeChannel {
     return this._on(type, filter, callback)
   }
-
+  /**
+   * Sends a message into the channel.
+   * @param args Arguments to send to the channel. Composed of a type, event, and payload.
+   * @param opts Options to send to the server.
+   * @returns
+   */
   async send(
-    payload: { type: string; [key: string]: any },
+    args: { type: string; event: string; payload?: any; [key: string]: any },
```

Comment:
Is event really a required field? Doesn't look like it if you use another type than broadcast

Reply:
This is also a potentially breaking change if you introduce a required field